### ISSUE TITLE
Do we plan to support inbound traffic parsing?

### ISSUE THREAD

**Issue by wenhuwang** [coroot/coroot-node-agent#105]
Currently, coroot-node-agent only parses network packets that the current node service actively requests for other services. However, in many scenarios, coroot-node-agent cannot be deployed to all nodes. Can it support parsing network packets requested by external services to build a more complete service topology?

**Comment by def** [comment]
Yes, we have plans to implement this. Here is a similar [issue](https://github.com/coroot/coroot/issues/281) in the main repo.

**Comment by wenhuwang** [comment]
How is the progress of this feature? I am interested in this feature and would like to contribute if possible.

**Comment by tripitakav** [comment]
Is has any new progress of this feature?

**Comment by StLeoX** [comment]
https://github.com/StLeoX/coroot-node-agent/commit/667b19c0ea0ba8820d1978268e8fc4c934ac646d

Here I design `l7_event`'s corresponding in-bound traffic `l7_event_ss` (`ss` means server-side, current process working as the server). But I meet the problem, I can't filter `l7_event_ss` by active TCP connections and FDs correctly.
Anyone could give me some advice? @wenhuwang @gravese

**Comment by evanzhang87** [comment]
Is there any progress on this issue now? 
I follow the current logic, If `enter_read` first then `enter_write`, I'll consider it as inbound traffic, it works, but I don't know if there are some exceptions.

**Comment by def** [comment]
https://github.com/coroot/coroot-node-agent/pull/310
